shifts.go

```diff
@@ -0,0 +1,141 @@
+package factorial
+
+import (
+	"encoding/json"
+	"net/url"
+)
+
+const (
+	shiftURL = "/api/v1/shifts"
+)
+
+// Shift keeps the basic information related
+// with shifts in Factorial
+type Shift struct {
+	ID           int    `json:"id"`
+	Day          int    `json:"day"`
+	Month        int    `json:"month"`
+	Year         int    `json:"year"`
+	ClockIn      string `json:"clock_in"`
+	ClockOut     string `json:"clock_out"`
+	EmployeeID   bool   `json:"employee_id"`
+	Observations string `json:"observations"`
+}
+
+// ClockInRequest will hold the basic information
+// needed for create a new shift (ClockIn) in Factorial
+type ClockInRequest struct {
+	Now        string `json:"now"`
+	EmployeeID int    `json:"employee_id"`
+}
+
+// ClockOutRequest will hold the basic information
+// needed for create a new shift (ClockOut) in Factorial
+type ClockOutRequest struct {
+	Now        string `json:"now"`
+	EmployeeID int    `json:"employee_id"`
+}
+
+// UpdateShiftRequest will hold the basic information
+// for update a given shift.
+// Restricted to the user's own shifts.
+type UpdateShiftRequest struct {
+	ClockIn      string `json:"clock_in"`
+	ClockOut     string `json:"clock_out"`
+	Observations string `json:"observations"`
+}
+
+// ClockIn creates a new Shift with the provided time and for the requested employee.
+// If an open Shift already exists, this endpoint will return an error with code 422.
+func (c Client) ClockIn(cin ClockInRequest) (Shift, error) {
```

Comment:
This ClockIn and ClockOut could be interested to test on the server example due is going to be our main core for the bot

Reply:
TRUE; I'll add a button to clockIn / clockOut :muscle: